DAO.sol

```diff
@@ -728,10 +736,10 @@ contract DAO is DAOInterface, Token, TokenSale {
     }
 
 
-    function addAllowedAddress(address _recipient) noEther external returns (bool _success) {
+    function changeAllowedRecipients(address _recipient, bool _allowed) noEther external returns (bool _success) {
```

Comment:
The UI is still going to be build (the autogenerated Mist UI is a temporary workaround). And this is only for the curators, not for the masses.